packages/core/src/parser/parser.ts

```diff
@@ -288,41 +335,84 @@ export class Parser extends CstParser {
      * an Equal sign and an operatorExpression on the right side.
      */
     private destructuringExpression = this.RULE(Rules.DESTRUCTURING_EXPRESSION, () => {
-        this.CONSUME(OpenRoundBracket);
+        const opening = this.CONSUME(OpenRoundBracket);
```

Comment:
would probably replace this in the error message just with the constant `(` as this cannot be anything else really